tendrl-api.spec

```diff
@@ -0,0 +1,60 @@
+Name: tendrl-api
+Version: 0.0.1
+Release: 1%{?dist}
+Summary: Collection of tendrl api extensions
+Group: Development/Languages
+License: LGPLv2+
+URL: https://github.com/Tendrl/tendrl-api
+Source0: %{name}-%{version}.tar.gz
+BuildArch: noarch
+
+BuildRequires: ruby
+
+Requires: httpd
```

Comment:
Doesn't require httpd. It can be run without httpd. There shouldn't be a hard dependency on httpd.